custom_components/tahoma/binary_sensor.py

```diff
@@ -58,6 +64,21 @@ def device_class(self):
             or None
         )
 
+    @property
+    def icon(self) -> Optional[str]:
+        """Return the icon to use in the frontend, if any."""
+
+        if self.device_class == DEVICE_CLASS_RAIN:
+            return "mdi:weather-rainy"
```

Comment:
can we use `hass` icons instead of `mdi`? I think they are all the same, but load faster.
`hass:weather-rainy`

Reply:
Is there any documentation around this? I am not aware of the icons changing to hass prefix?

Reply:
I can't find where I've read this... Let's keep mdi.